src/components/NcActions/NcActions.vue

```diff
@@ -1769,6 +1839,20 @@ export default {
 		 */
 		const menuActions = actions.filter(action => !inlineActions.includes(action))
 
+		if (this.submenuStack.length) {
+			const backButton = h(NcActionButton, {
+				on: {
+					click: () => {
+						this.popSubmenu()
+					},
```

Comment:
Since we have an arrow navigation ⬆️ ⬇️ , would it make sense to add ⬅️ listener to 'Back' button, and ➡️ listener to 'isMenu' buttons?

Reply:
Yeah, I would agree to use arrows rather escape (I would expect it to close the menu itself and not just submenus)

Reply:
I agree to add arrows, but I'd like to keep <kbd>Escape</kbd>. Unlike a classic multi-level menu which literally opens side by side, Nextcloud's submenus act more like a new menu. 

For example, in this state when "Next week" is under focus, it's quite non-obvious to me that "Arrow left" can close the menu. There is nothing "on the left", no arrows on the button. 

![image](https://github.com/user-attachments/assets/7890c977-08f3-4c5a-a7c9-6a087d760972)

Reply:
A bit unusual to have 'left' keydown only on 'Back button' (works in Windows - but they open submenu next to menu, not replacing it), but more comfortable anyway